packages/smart-wallet/src/smartWallet.js

```diff
@@ -315,10 +356,30 @@ export const prepareSmartWallet = (baggage, shared) => {
         .returns(M.promise()),
       publishCurrentState: M.call().returns(),
       watchPurse: M.call(M.eref(PurseShape)).returns(M.promise()),
+      repairUnwatchedSeats: M.call().returns(),
+      updateStatus: M.call(M.any()).returns(),
+      addContinuingOffer: M.call(
+        M.or(M.number(), M.string()),
+        AmountShape,
+        M.remotable('InvitationMaker'),
+        M.or(M.record(), M.undefined()),
+      ).returns(M.promise()),
+      purseForBrand: M.call(BrandShape).returns(M.promise()),
+      logWalletInfo: M.call(M.any()).returns(),
+      logWalletError: M.call(M.any()).returns(),
+      // XXX is there a better guard for a bigMapStore?
```

Comment:
```suggestion
      // XXX is there a tighter shape for a bigMapStore than M.any()?
```

fwiw I don't think so